internal/services/v1/schema.go

```diff
@@ -46,15 +45,21 @@ type schemaServer struct {
 }
 
 func (ss *schemaServer) ReadSchema(ctx context.Context, in *v1.ReadSchemaRequest) (*v1.ReadSchemaResponse, error) {
-	readRevision, _ := consistency.MustRevisionFromContext(ctx)
-	ds := datastoremw.MustFromContext(ctx).SnapshotReader(readRevision)
+	// Schema is already read from the head revision.
```

Comment:
hrm, this comment assumes the reader has context about the previous implementation. I'd clarify that this method does not have consistency specified and thus we need to explicitly get `head` revision as it won't be available in the context.

Reply:
Sorry; should have said "always". I've fixed